etl.js

```diff
@@ -146,7 +145,39 @@ Etl.upsertChecklistItem = async function _upsertChecklistItem(item) {
     );
     projectMeta.amount = item._amount;
     store.set(`${ISSUE_TO_ITEM}:${item.id}:project`, projectMeta);
-    await sleep(SLEEP);
+  }
+
+  let secondaryAdminID = "5ff85abd2b962872d01fe3bf";
+  let fallbackOwnerField = card.customFieldItems.find(function (card) {
+    return card.idCustomField === secondaryAdminID;
+  });
+  let fallbackOwner = fallbackOwnerField?.value?.text;
+  let isExpectedFallbackOwner = fallbackOwner === projectMeta.fallback_owner;
+  if (fallbackOwner && !isExpectedFallbackOwner) {
+    await gh.projects.setFallbackOwner(
+      projectMeta.projectItemNodeId,
+      fallbackOwner
+    );
+    projectMeta.fallback_owner = fallbackOwner;
+    store.set(`${ISSUE_TO_ITEM}:${item.id}:project`, projectMeta);
+  }
+  let fallbackOwnerId = "";
+  let ownerArray = card.idMembers;
+  if (fallbackOwner) {
+    fallbackOwnerId = Transform.usernameToId(fallbackOwner);
+  }
+
+  let ownerIdArray = ownerArray.filter(function (ownerId) {
+    return ownerId !== fallbackOwnerId;
+  });
+  let ownerId = ownerIdArray.toString();
```

Comment:
This doesn't make sense... ownerId is supposed to be a string of ids? `"abc123,def345,ghi678"`?

Did you mean to use `.find()` rather than `.filter()` and just get the first match?